src/app/modules/community-core/services/library-service/library.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { LibraryConfig } from './library.config';
+import { HttpClient } from '@angular/common/http';
+import { environment } from 'src/environments/environment';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LibraryService {
+
+  constructor(private Http: HttpClient) { }
+
+
+  // To get the Filter List
+  getFilterList() {
+    return this.Http.get(environment.base_url + LibraryConfig.filterList);
```

Comment:
Why are we not using the npm service which we created??

Reply:
changed to npm usage